### PR TITLE
Updated the titles in the JSONs

### PR DESCRIPTION
**Pull request by bikashtudu**
|PR checklist|Status| 
|-----|:------:| 
|Is issue created in GitHub?|[Yes](https://github.com/maqsoftware/Simple-Class/issues/159)| 
|Are the changes replicated across different locales?|No| 
|Is code analysis task executed and issues resolved in the Java/Kotlin code?|NA| 
|Is code analysis task executed and issues resolved in the CPP/native code?|NA| 
|Is the version code set in the AndroidManifest.xml or build.gradle file to 1 more than the version which has already been published?|NA| 
|Is code commenting done properly for the changes made?|NA| 
|Is correct variable naming convention followed?|NA| 
|Is auto formatting shortcut Ctrl+Alt+Shift+L executed in all files?|NA| 
|Are the message strings used in the application taken from strings.xml file?|NA| 
|Are the permissions which are not required removed from AndroidManifest.java file?|NA| 
|Is the _uses-feature_ tag used for only must have feature in AndroidManifest.java file?|NA| 
|Are the generated files correctly configured in .gitignore?|NA| 
|Are the release signing configurations removed from the gradle files?|NA| 
|Is commented code removed from all files?|NA| 
|Are updated images compressed (< 100 KB)?|NA| 
|Ensured there are no typos in any string. For example: Variable/method names, hardcoded strings, commit messages, PR titles, issues, code comments?|Yes|

### CODE REVIEW

File: goa/Resources/res/config/grammar_game_map_hi.json [hunk 1]
```diff
@@ -608,8 +607,8 @@
   },
   {
     "name": "pop",
-    "title": "बादलों के माध्यम से उड़ो$#$Pop",
-	"_title_comment":"बादलों के माध्यम से उड़ो",
+    "title": "बादलों मै उड़ो$#$Fly through Clouds",
```
Comment (ankitk20) [inline goa/Resources/res/config/grammar_game_map_hi.json]: में and not मैं

File: goa/Resources/res/config/grammar_game_map_hi.json [hunk 2]
```diff
@@ -797,8 +796,8 @@
   },
   {
     "name": "pillar",
-    "title": "केक परत$#$Pillar",
-	"_title_comment": "केक परत",
+    "title": "केक परत$#$Layer the Cake",
```
Comment (ankitk20) [inline goa/Resources/res/config/grammar_game_map_hi.json]: Cake Layer

File: goa/Resources/res/config/shapes_game_map_hi.json [hunk 3]
```diff
@@ -516,8 +516,8 @@
   },
   {
     "name": "shop",
-    "title": "किराने की दुकान$#$Shop",
-	"_title_comment":"किराने की दुकान",
+    "title": "किराने की दुकान$#$Grocery Shop",
```
Comment (ankitk20) [inline goa/Resources/res/config/shapes_game_map_hi.json]: Make Hindi name to sabzi mandi

File: goa/Resources/res/config/shapes_game_map_hi.json [hunk 4]
```diff
@@ -558,8 +558,8 @@
   },
   {
     "name": "Units",
-    "title": "एक पिज्जा बनाओ$#$Units",
-	"_title_comment":"एक पिज्जा बनाओ",
+    "title": "पिज्जा बनाओ$#$Make a Pizza",
```
Comment (ankitk20) [inline goa/Resources/res/config/shapes_game_map_hi.json]: it should be pizza and not pijja there should be a dot below ja

File: goa/Resources/res/config/shapes_game_map_hi.json [hunk 5]
```diff
@@ -726,8 +726,8 @@
   },
   {
     "name": "Bounce",
-    "title": "आइसक्रीम$#$Ice-Cream",
-	"_title_comment":"आइसक्रीम",
+    "title": "आइसक्रीम$#$Drop the Ice-cream",
```
Comment (ankitk20) [inline goa/Resources/res/config/shapes_game_map_hi.json]: Ice-cream girao

File: goa/Resources/res/config/writing_game_map_en.json [hunk 6]
```diff
@@ -111,7 +111,7 @@
   },
   {
     "name": "cardraw",
-    "title": "Go Home",
+    "title": "Letter Writing",
```
Comment (ankitk20) [inline goa/Resources/res/config/writing_game_map_en.json]: Make it Alphabet Writing

File: goa/Resources/res/config/writing_game_map_hi.json [hunk 7]
```diff
@@ -112,8 +112,8 @@
   },
   {
     "name": "cardraw",
-    "title": "घर जाओ$#$Car Draw",
-	"_title_comment":"घर जाओ",
+    "title": "अक्षर बनाओ$#$Letter Writing",
```
Comment (ankitk20) [inline goa/Resources/res/config/writing_game_map_hi.json]: Alphabet Writing

File: scripts/get_titles_mismatches.py [hunk 8]
```diff
@@ -0,0 +1,125 @@
+import json
```
Comment (ankitk20) [inline scripts/get_titles_mismatches.py]: Create a folder in Pehla-School repo with name Utilities and move all Python scripts inside it. Do the same for Simple Class project also.

File: scripts/get_titles_mismatches.py [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+import xlsxwriter
+
+
+def get_title_miss_matches(ref_json, check_json, xlsx_file):
+    """
+    This function create an excel sheet which contains all the miss-matched
```
Comment (ankitk20) [inline scripts/get_titles_mismatches.py]: Make sheet no., col. name, Excel file name configurable by adding a global variable on the top. If same value is reused many times then it is always better to create a variable for it.

File: scripts/get_titles_mismatches.py [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+        for each_check_object in check_json_data:
+            check_object_name = each_check_object['name']
+            check_object_title = each_check_object['title'].split('$#$')
+            ref_object = ref_json_dict[check_object_name]
+            ref_object_title = ref_object['title']
+            if ref_object_title != check_object_title[1]:
+                sheet.write(rowCount, 0, check_object_name)
+                sheet.write(rowCount, 1, ref_object_title)
+                sheet.write(rowCount, 2, check_object_title[1])
+                rowCount = rowCount+1
```
Comment (ankitk20) [inline scripts/get_titles_mismatches.py]: Check formatting is correct on this line

File: scripts/get_titles_mismatches.py [hunk 11]
```diff
@@ -0,0 +1,125 @@
...
+        for each_object in ref_json_data:
+            ref_json_dict[each_object['name']] = each_object
+    rowCount = rowCount+1
+    with open(check_json, 'r', encoding='utf-8-sig') as check_file:
+        check_json_data = json.load(check_file)
+        length = len(check_json_data)
+        for i in range(length):
+            each_check_object = check_json_data[i]
+            check_object_name = each_check_object['name']
+            check_object_title = each_check_object['title'].split('$#$')
```
Comment (ankitk20) [inline scripts/get_titles_mismatches.py]: Create a delim var rather than repeating same string

File: scripts/json_refractor.py [hunk 12]
```diff
@@ -0,0 +1,78 @@
+import json
+import xlrd
+
+
+def replace_title_ref_(ref_excel, curr_json, new_json, json_locale='en', title_index=1, sheet_name='Sheet 1'):
+    """
+    This function create a new json file which contains english part of the
+    title referred from ref_excel file.
+    ref_excel: excel file from where name,title for each object will be referred
```
Comment (ankitk20) [inline scripts/json_refractor.py]: Product names should start with capital letter

File: scripts/json_refractor.py [hunk 13]
```diff
@@ -0,0 +1,78 @@
+import json
```
Comment (ankitk20) [inline scripts/json_refractor.py]: Typo in file name

File: scripts/json_refractor.py [hunk 14]
```diff
@@ -0,0 +1,78 @@
...
+import xlrd
```
Comment (ankitk20) [inline scripts/json_refractor.py]: Mention on the top of every script that what it does in brief

File: scripts/json_refractor.py [hunk 15]
```diff
@@ -0,0 +1,78 @@
...
+                if json_locale != 'en':
+                    check_object_title = each_check_object['title'].split('$#$')
+                    check_object_title[title_index] = ref_object_title
+                    new_check_object_title = '$#$'.join(check_object_title)
+                else:
+                    new_check_object_title = ref_object_title
+                each_check_object['title'] = new_check_object_title
+            check_json_data[i] = each_check_object
+    with open(new_json, 'w', encoding='utf-8') as new_jsonfile:
+        json.dump(check_json_data, new_jsonfile, ensure_ascii=False, indent=2)
```
Comment (ankitk20) [inline scripts/json_refractor.py]: Comments are missing

**Review by ankitk20 — CHANGES_REQUESTED** [review]

File: goa/Resources/res/config/shapes_game_map_hi.json [hunk 16]
```diff
@@ -516,8 +516,8 @@
   },
   {
     "name": "shop",
-    "title": "किराने की दुकान$#$Shop",
-	"_title_comment":"किराने की दुकान",
+    "title": "सब्जी मंडी$#$Grocery Shop",
```
Comment (ankitk20) [inline goa/Resources/res/config/shapes_game_map_hi.json]: Sab**z**i and not Sab**j**i

File: Utilities/get_titles_mismatches.py [hunk 17]
```diff
@@ -0,0 +1,142 @@
+"""
+This script has functions to create Excel for mismatched titles, replace titles from
+JSON reference, and Excel with all the titles from Hindi JSON file
```
Comment (ankitk20) [inline Utilities/get_titles_mismatches.py]: With Alankruta's help please fix this. Intent is not getting conveyed properly.

File: Utilities/get_titles_mismatches.py [hunk 18]
```diff
@@ -0,0 +1,142 @@
...
+"""
+import json
+import xlsxwriter
+
+
+def get_title_miss_matches(ref_json, check_json, xlsx_file, sheet_name="Sheet 1"):
+    """
+    This function create an excel sheet which contains all the miss-matched
```
Comment (ankitk20) [inline Utilities/get_titles_mismatches.py]: Product name first letter should be capital

Comment (ankitk20) [reply Utilities/get_titles_mismatches.py]: Spelling of mismatched is incorrect

File: Utilities/get_titles_mismatches.py [hunk 19]
```diff
@@ -0,0 +1,142 @@
...
+            check_object_name = each_check_object[ident_key]
+            check_object_title = each_check_object[target_key].split(dem_char)
+            ref_object = ref_json_dict[check_object_name]
+            ref_object_title = ref_object[target_key]
+            if ref_object_title != check_object_title[1]:
+                sheet.write(rowCount, 0, check_object_name)
+                sheet.write(rowCount, 1, ref_object_title)
+                sheet.write(rowCount, 2, check_object_title[1])
+                rowCount = rowCount+1
+            # sheet.write(rowCount, 3, check_object_title[0])
```
Comment (ankitk20) [inline Utilities/get_titles_mismatches.py]: Removed commented code

File: Utilities/get_titles_mismatches.py [hunk 20]
```diff
@@ -0,0 +1,142 @@
...
+    with open(check_json, 'r', encoding='utf-8-sig') as check_file:
+        check_json_data = json.load(check_file)
+        length = len(check_json_data)
+        for i in range(length):
+            each_check_object = check_json_data[i]
+            check_object_name = each_check_object[ident_key]
+            check_object_title = each_check_object[target_key].split(demchar)
+            ref_object = ref_json_dict[check_object_name]
+            ref_object_title = ref_object[target_key]
+            new_check_object_title = check_object_title[0]+ident_key+ref_object_title
```
Comment (ankitk20) [inline Utilities/get_titles_mismatches.py]: Check formatting

File: Utilities/get_titles_mismatches.py [hunk 21]
```diff
@@ -0,0 +1,142 @@
...
+    This function create a new json file which contains english part of the
+    title referred from ref_json file.
+    ref_json: JSON file from where name,title for each object will be referred
+    check_json: JSON file which titles needed to be checked
+    dest_file: destination JSON file which will store contain updated objects
+
+    Example:
+        replace_title_ref_('alphabet_game_map_en.json', 'alphabet_game_map_hi.json', 'alphabet_game_map_hi_new.json')
+    """
+    demchar = '$#$'
```
Comment (ankitk20) [inline Utilities/get_titles_mismatches.py]: demchar is below written as dem_char. Please maintain consistency in naming variables.

File: Utilities/get_titles_mismatches.py [hunk 22]
```diff
@@ -0,0 +1,142 @@
...
+        for each_object in ref_json_data:
+            ref_json_dict[each_object[ident_key]] = each_object
+    sheet = wb.add_worksheet(sheet_name)
+
+    rowCount = 0
+    sheet.write(rowCount, 0, 'name')
+    sheet.write(rowCount, 1, 'en_json_title')
+    sheet.write(rowCount, 2, 'hi_json_title_eng')
+    sheet.write(rowCount, 3, 'hi_json_title_hi')
+    rowCount = rowCount+2
```
Comment (ankitk20) [inline Utilities/get_titles_mismatches.py]: Check formatting

File: Utilities/get_titles_mismatches.py [hunk 23]
```diff
@@ -0,0 +1,142 @@
...
+        for each_check_object in check_json_data:
+            check_object_name = each_check_object[ident_key]
+            check_object_title = each_check_object[target_key].split(dem_char)
+            ref_object = ref_json_dict[check_object_name]
+            ref_object_title = ref_object[target_key]
+            sheet.write(rowCount, 0, check_object_name)
+            sheet.write(rowCount, 1, ref_object_title)
+            sheet.write(rowCount, 2, check_object_title[1])
+            sheet.write(rowCount, 3, check_object_title[0])
+            rowCount = rowCount+1
```
Comment (ankitk20) [inline Utilities/get_titles_mismatches.py]: Check formatting

File: Utilities/get_titles_mismatches.py [hunk 24]
```diff
@@ -0,0 +1,142 @@
...
+    wb.close()
+
+get_titles_name_json('original/alphabet_game_map_en.json', 'original/alphabet_game_map_hi.json', 'excels/alphabet_titles.xlsx')
+get_titles_name_json('original/grammar_game_map_en.json', 'original/grammar_game_map_hi.json', 'excels/grammar_titles.xlsx')
+get_titles_name_json('original/shapes_game_map_en.json', 'original/shapes_game_map_hi.json', 'excels/shapes__titles.xlsx')
+get_titles_name_json('original/writing_game_map_en.json', 'original/writing_game_map_hi.json', 'excels/writing_titles.xlsx')
```
Comment (ankitk20) [inline Utilities/get_titles_mismatches.py]: As comment given previously... use for loop and append locale string in the file name. Store the locale name list in a array. Evaluate whether make it global will be better.

File: Utilities/json_refactor.py [hunk 25]
```diff
@@ -0,0 +1,93 @@
...
+    target_key = 'title'
+    # Read the Excel sheet to create a dictionary
+    wb = xlrd.open_workbook(ref_excel)
+    sheet = wb.sheet_by_name(sheet_name)
+    rowCount = sheet.nrows
+    colCount = sheet.ncols
+    # Get the column index of identity and new title column in Excel
+    name_index = -1
+    newtitle_index = -1
+    for i in range(colCount):
```
Comment (ankitk20) [inline Utilities/json_refactor.py]: Why did not jump from snake case to camel case convention?

File: Utilities/json_refactor.py [hunk 26]
```diff
@@ -0,0 +1,93 @@
...
+    # identity key(column name/key) for the object in the JSON and Excel
+    ident_key = 'name'
+    # column name for the new title
+    src_key = 'new_title'
+    # key of the target that needed to be changed
+    target_key = 'title'
+    # Read the Excel sheet to create a dictionary
+    wb = xlrd.open_workbook(ref_excel)
+    sheet = wb.sheet_by_name(sheet_name)
+    rowCount = sheet.nrows
```
Comment (ankitk20) [inline Utilities/json_refactor.py]: Why not use snake case here as well?

File: Utilities/json_refactor.py [hunk 27]
```diff
@@ -0,0 +1,93 @@
...
+            if check_object_name in newtitle_dict:
+                ref_object_title = newtitle_dict[check_object_name]
+                if json_locale != 'en':
+                    check_object_title = each_check_object[target_key].split(dem_char)
+                    check_object_title[title_index] = ref_object_title
+                    new_check_object_title = dem_char.join(check_object_title)
+                else:
+                    new_check_object_title = ref_object_title
+                each_check_object[target_key] = new_check_object_title
+            check_json_data[i] = each_check_object
```
Comment (ankitk20) [inline Utilities/json_refactor.py:72]: Check indentation in else block

File: Utilities/json_refactor.py [hunk 28]
```diff
@@ -0,0 +1,93 @@
...
+replace_title_ref_(mismatched_titles_excel, 'original/shapes_game_map_en.json', 'en_updated/shapes_game_map_en.json',  sheet_name='shapes')
+replace_title_ref_(mismatched_titles_excel, 'original/writing_game_map_hi.json', 'en_updated/writing_game_map_hi.json', 'hi', sheet_name='writing')
+replace_title_ref_(mismatched_titles_excel, 'original/writing_game_map_en.json', 'en_updated/writing_game_map_en.json',  sheet_name='writing')
+
+# Update the Hindi titles in the Hindi JSONs files
+hindi_titles_excel = 'game_titles.xlsx'
+replace_title_ref_(hindi_titles_excel, 'en_updated/alphabet_game_map_hi.json', 'hi_updated/alphabet_game_map_hi.json', 'hi', title_index=0,  sheet_name='alphabet')
+replace_title_ref_(hindi_titles_excel, 'en_updated/grammar_game_map_hi.json', 'hi_updated/grammar_game_map_hi.json', 'hi', title_index=0, sheet_name='grammar')
+replace_title_ref_(hindi_titles_excel, 'en_updated/shapes_game_map_hi.json', 'hi_updated/shapes_game_map_hi.json', 'hi', title_index=0, sheet_name='shapes')
+replace_title_ref_(hindi_titles_excel, 'en_updated/writing_game_map_hi.json', 'hi_updated/writing_game_map_hi.json', 'hi', title_index=0, sheet_name='writing')
```
Comment (ankitk20) [inline Utilities/json_refactor.py]: Use for loop and append locale string to file names. Store the locale in a array. This comment is applicable from line 73 to 93.

**Review by ankitk20 — CHANGES_REQUESTED** [review]

File: Utilities/get_titles_mismatches.py (hunk 18 above)
Comment (alankrutap01) [reply Utilities/get_titles_mismatches.py]: function 'creates'
mismatched what?

File: Utilities/get_titles_mismatches.py [hunk 29]
```diff
@@ -0,0 +1,142 @@
...
+import json
+import xlsxwriter
+
+
+def get_title_miss_matches(ref_json, check_json, xlsx_file, sheet_name="Sheet 1"):
+    """
+    This function create an excel sheet which contains all the miss-matched
+    titles in the check_json file.
+    ref_json: JSON file from where name,title for each object will be referred
+    check_json: JSON file which titles needed to be checked
```
Comment (alankrutap01) [inline Utilities/get_titles_mismatches.py]: where instead of which, need to be

File: Utilities/get_titles_mismatches.py [hunk 30]
```diff
@@ -0,0 +1,142 @@
...
+    xlsx_file: Excel file that will be contains all the mismatched titles
```
Comment (alankrutap01) [inline Utilities/get_titles_mismatches.py]: that will contain

File: Utilities/get_titles_mismatches.py [hunk 31]
```diff
@@ -0,0 +1,142 @@
...
+                sheet.write(rowCount, 1, ref_object_title)
+                sheet.write(rowCount, 2, check_object_title[1])
+                rowCount = rowCount+1
+            # sheet.write(rowCount, 3, check_object_title[0])
+    wb.close()
+
+
+def replace_title_ref_(ref_json, check_json, dest_file):
+    """
+    This function create a new json file which contains english part of the
```
Comment (alankrutap01) [inline Utilities/get_titles_mismatches.py]: creates

File: Utilities/get_titles_mismatches.py [hunk 32]
```diff
@@ -0,0 +1,142 @@
...
+    title referred from ref_json file.
+    ref_json: JSON file from where name,title for each object will be referred
+    check_json: JSON file which titles needed to be checked
```
Comment (alankrutap01) [inline Utilities/get_titles_mismatches.py]: where instead of which, need to be

File: Utilities/get_titles_mismatches.py [hunk 33]
```diff
@@ -0,0 +1,142 @@
...
+    dest_file: destination JSON file which will store contain updated objects
```
Comment (alankrutap01) [inline Utilities/get_titles_mismatches.py]: stores or contains?

File: Utilities/get_titles_mismatches.py [hunk 34]
```diff
@@ -0,0 +1,142 @@
...
+            each_check_object[target_key] = new_check_object_title
+            check_json_data[i] = each_check_object
+    # Create a new JSON file from the file
+    with open(dest_file, 'w', encoding='utf-8') as destfile:
+        json.dump(check_json_data, destfile, ensure_ascii=False, indent=2)
+
+
+def get_titles_name_json(ref_json, check_json, xlsx_file, sheet_name="Sheet 1"):
+    """
+    This function create an Excel sheet which contain name and title from ref_json, english and hindi separated title
```
Comment (alankrutap01) [inline Utilities/get_titles_mismatches.py]: creates, contains, titles

File: Utilities/get_titles_mismatches.py [hunk 35]
```diff
@@ -0,0 +1,142 @@
...
+    from the check_json file
+    ref_json: file from where name,title for each object will be referred
+    check_json: file which titles needed to be checked
```
Comment (alankrutap01) [inline Utilities/get_titles_mismatches.py]: Check previous review comment

File: Utilities/get_titles_mismatches.py [hunk 36]
```diff
@@ -0,0 +1,142 @@
...
+    xlsx_file: excel file that will be contains the mentioned field from above
```
Comment (alankrutap01) [inline Utilities/get_titles_mismatches.py]: contain, the above mentioned field

File: Utilities/json_refactor.py [hunk 37]
```diff
@@ -0,0 +1,93 @@
+"""
+This script update the hindi and english titles of the games in the JSON
```
Comment (alankrutap01) [inline Utilities/json_refactor.py]: updates

File: Utilities/json_refactor.py [hunk 38]
```diff
@@ -0,0 +1,93 @@
...
+"""
+import json
+import xlrd
+
+
+def replace_title_ref_(ref_excel, curr_json, new_json, json_locale='en', title_index=1, sheet_name='Sheet 1'):
+    """
+    This function create a new JSON file which contains English part of the
```
Comment (alankrutap01) [inline Utilities/json_refactor.py]: creates

File: Utilities/json_refactor.py [hunk 39]
```diff
@@ -0,0 +1,93 @@
...
+    title referred from ref_excel file.
+    ref_excel: Excel file from where name,title for each object will be referred
+    curr_json: JSON file in which titles needed to be replace
```
Comment (alankrutap01) [inline Utilities/json_refactor.py]: Check previous review comment

File: Utilities/json_refactor.py [hunk 40]
```diff
@@ -0,0 +1,93 @@
...
+    new_json: destination file which will store contain updated titles
```
Comment (alankrutap01) [inline Utilities/json_refactor.py]: Check previous review comment

File: goa/Resources/res/config/shapes_game_map_hi.json [hunk 41]
```diff
@@ -558,8 +558,8 @@
   },
   {
     "name": "Units",
-    "title": "एक पिज्जा बनाओ$#$Units",
-	"_title_comment":"एक पिज्जा बनाओ",
+    "title": "पिज़्ज़ा बनाओ$#$Make a Pizza",
+    "_title_comment": "एक पिज्जा बनाओ",
```
Comment (alankrutap01) [inline goa/Resources/res/config/shapes_game_map_hi.json:562]: Keep consistency for Pizza spelling

File: Utilities/get_titles_mismatches.py (hunk 17 above)
Comment (alankrutap01) [reply Utilities/get_titles_mismatches.py]: This script has functions to create Excel sheets for the mismatched titles, to replace titles from JSON reference file and also contains an Excel function with titles from Hindi JSON file

File: goa/Resources/res/config/shapes_game_map_hi.json (hunk 16 above)
Comment (bikashtudu) [reply goa/Resources/res/config/shapes_game_map_hi.json]: Fixed the spelling

File: Utilities/get_titles_mismatches.py (hunk 18 above)
Comment (bikashtudu) [reply Utilities/get_titles_mismatches.py]: mismatched titles, titles is in next title

File: Utilities/get_titles_mismatches.py (hunk 20 above)
Comment (bikashtudu) [reply Utilities/get_titles_mismatches.py]: Formatted the code.

File: Utilities/get_titles_mismatches.py (hunk 24 above)
Comment (bikashtudu) [reply Utilities/get_titles_mismatches.py]: Updated the code as per the comment.

File: goa/Resources/res/config/shapes_game_map_hi.json (hunk 41 above)
Comment (bikashtudu) [reply goa/Resources/res/config/shapes_game_map_hi.json:562]: पिज़्ज़ा is the correct spelling.

File: Utilities/get_titles_mismatches.py (hunk 21 above)
Comment (bikashtudu) [reply Utilities/get_titles_mismatches.py]: updated to dem_char

File: goa/Resources/res/config/shapes_game_map_hi.json (hunk 41 above)
Comment (ankitk20) [reply goa/Resources/res/config/shapes_game_map_hi.json:562]: title_comment pizza spelling

**Review by ankitk20 — APPROVED** [review]

**Pull request merged**